api/admin/geographic_validator.py

```diff
@@ -74,27 +74,31 @@ def validate_geographic_areas(self, values, db):
                         # the registry won't recognize the name of the place to which it corresponds.
                         registry_response = self.find_location_through_registry(formatted, db)
                         if registry_response:
-                            locations["CA"].append(formatted);
+                            locations["CA"].append(formatted)
                         else:
                             return UNKNOWN_LOCATION.detailed(_('Unable to locate "%(value)s" (%(formatted)s).  Try entering the name of a larger area.', value=value, formatted=formatted))
                     except:
                         return UNKNOWN_LOCATION.detailed(_('"%(value)s" is not a valid Canadian zipcode.', value=value))
                 elif len(value.split(", ")) == 2:
                     # Is it in the format "[city], [state abbreviation]" or "[county], [state abbreviation]"?
                     city_or_county, state = value.split(", ")
-                    if us_search.by_city_and_state(city_or_county, state):
-                        locations["US"].append(value);
-                    elif len([x for x in us_search.query(state=state, returns=None) if x.county == city_or_county]):
-                        locations["US"].append(value);
-                    else:
-                        # Flag this as needing to be checked with the registry
+                    try:
+                        if us_search.by_city_and_state(city_or_county, state):
+                            locations["US"].append(value)
+                        elif len([x for x in us_search.query(state=state, returns=None) if x.county == city_or_county]):
+                            locations["US"].append(value)
+                        else:
+                            # Flag this as needing to be checked with the registry
+                            flagged = True
+                    except ValueError:
```

Comment:
`uszipcode` added fuzzy searching that can't be toggled off. An invalid city/state search will attempt to find something while raising a `ValueError`.